src/main/java/jp/ne/paypay/model/Payment.java

```diff
@@ -336,6 +371,8 @@ public String toString() {
     sb.append("    metadata: ").append(toIndentedString(metadata)).append("\n");
     sb.append("    expiresAt: ").append(toIndentedString(expiresAt)).append("\n");
     sb.append("    amountDescription: ").append(toIndentedString(amountDescription)).append("\n");
+    sb.append("    paymentMethodType: ").append(toIndentedString(paymentMethodType)).append("\n");
+    sb.append("    paymentMethodId: ").append(toIndentedString(paymentMethodId)).append("\n");
```

Comment:
Lets not print the payment method id as a security stand point.
If we missed in get payment methods api, lets remove it from toString() there also

Reply:
I see. I removed this, and id in PaymentMethod.java.